paymentsheet/build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: 'app.cash.paparazzi'
 
 dependencies {
     implementation project(":link")
-    implementation project(":payments-core")
+    api project(":payments-core")
```

Comment:
Is this intentional? Do we really need this in this PR?

Reply:
Oh, looks like a rebase issue to me. This change is already in `tillh/decoupling-beta-breaking-changes`. I’ll rebase again.